src/test/regress/expected/ch_benchmarks_create_load.out

```diff
@@ -148,72 +154,144 @@ SELECT create_distributed_table('order_line','ol_w_id');
 
 (1 row)
 
+SELECT grant_table_to_regularuser('"ch benchmarks"."order_line"');
```

Comment:
Ok, I think I can see why we ended up in this way. Because while regclass is resolved, Postgres already throws the error:
```SQL
select create_distributed_table('sc1.test', 'a');                                  
ERROR:  permission denied for schema sc1
LINE 1: select create_distributed_table('sc1.test', 'a');
```

Which means that we cannot easily override  `create_distributed_table` to work on a table that has not been granted to the user.

So, I give up for now. 

Still, I'm not %100 sure that merging this PR is also good as it adds these to external steps.